cfgov/v1/admin.py

```diff
@@ -1,7 +1,28 @@
 from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin
+from django.contrib.auth.models import User
 from models.snippets import Contact
 
+from v1.email import send_password_reset_email
+
+
 @admin.register(Contact)
 class ContactAdmin(admin.ModelAdmin):
     pass
 
+
+admin.site.unregister(User)
+
+@admin.register(User)
+class UserAdmin(UserAdmin):
+    actions = UserAdmin.actions + ['send_password_reset_email']
+
+    def send_password_reset_email(self, request, queryset):
+        for user in queryset:
+            send_password_reset_email(user.email, request)
```

Comment:
Thanks, that's good feedback. There's lots of ways this is suboptimal: what if one of the sends fails for some reason, then it breaks halfway through, then you lose track of who was mailed? Also as you mention if there are too many emails selected, the request might take too long and cause the server to time out. Using [send_mass_mail](https://docs.djangoproject.com/en/1.8/topics/email/#send-mass-mail) instead would require a significant rewrite of the email sending code (actually implementing it, instead of just piggybacking on Django's [PasswordResetForm](https://github.com/django/django/blob/1.8.14/django/contrib/auth/forms.py#L195) as I do now). I thought this was a reasonable balance for what is really just a helper admin utility.